src/SeqCli/Config/SeqCliConfig.cs

```diff
@@ -52,7 +52,11 @@ public static void Write(SeqCliConfig data)
     }
 
     public SeqCliConnectionConfig Connection { get; set; } = new SeqCliConnectionConfig();
-    public SeqCliOutputConfig Output { get; set; } = new SeqCliOutputConfig();
+    public SeqCliOutputConfig Output { get; set; } = new();
+    public SeqCliStorageConfig Storage { get; set; } = new();
+    public SeqCliDiagnosticConfig Diagnostics { get; set; } = new();
+    public SeqCliApiConfig Api { get; set; } = new();
```

Comment:
Perhaps these three would be best grouped under a `Forwarder` sub-object? Settings like `api.listenUri` are a little vague when viewed from the general `seqcli` perspective, they'd be easier to read/understand as `forwarder.api.listenUri` :thinking: